tests/include/test/drivers/key_management.h

```diff
@@ -30,6 +30,10 @@ typedef struct {
     /* Count the amount of times one of the key management driver functions
      * is called. */
     unsigned long hits;
+    /* Subset of hits which only counts public key export operations */
+    unsigned long hits_export_public_key;
```

Comment:
My initial reasoning was that adding the field would result in fewer contextual conflicts in the future, but as you point out, if we add the field, we should actually use it - otherwise we're leaving things in an inconsistent state that is likely to cause a lot more head scratching that a simple contextual conflict in the future. Thanks for catching this.